### PR TITLE
fix sft loss promlem

### PR DESCRIPTION
**Pull request by jiangchengSilent**
according to discussion in 
https://github.com/microsoft/DeepSpeedExamples/issues/301
https://github.com/microsoft/DeepSpeedExamples/pull/762/files

### CODE REVIEW

File: finetune/utils/data/data_utils.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def get_raw_dataset(dataset_name, output_path, seed, local_rank):
             and os.path.isfile(chat_path + "/data/eval.json")
         ):
             raise RuntimeError(
-                "Please check both the train.json and eval.json files in your local directory."
+                f"Please check both the train.json and eval.json files in your local directory."
```
Comment (loofahcus) [inline finetune/utils/data/data_utils.py:39]: why f-string here?

Comment (jiangchengSilent) [reply finetune/utils/data/data_utils.py]: copy from a former version before going public and not refreshed by pull request
removed

File: finetune/utils/data/raw_datasets.py [hunk 2]
```diff
@@ -154,6 +154,11 @@ def get_eval_data(self):
         if self.raw_datasets["eval"] is not None:
             return self.raw_datasets["eval"]
         return None
+    
+    def get_prompt(self, sample):
+        if sample["prompt"] is not None:
+            return " " + sample["prompt"]
```
Comment (loofahcus) [inline finetune/utils/data/raw_datasets.py:160]: why do we add an extra space here?

Comment (jiangchengSilent) [reply finetune/utils/data/raw_datasets.py:160]: this is a routine inherited from data_utils from ds-chat

Comment (loofahcus) [reply finetune/utils/data/raw_datasets.py:160]: got it

**Pull request merged**